agent/custom/bbc_connection_manager.py

```
@@ -115,6 +192,12 @@ def _permanent_callback_loop(self, server_sock: socket.socket):
                 
                 msg = json.loads(data.decode('utf-8'))
                 
+                # 处理关闭信号
+                if msg.get('event') == '__shutdown__':
+                    mfaalog.info(f"[BbcConnectionManager] 收到关闭信号，退出回调线程")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)